AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Storage/SQLite/ModelSchema+SQLite.swift

```diff
@@ -118,10 +118,13 @@ extension ModelSchema {
     /// Create SQLite indexes corresponding to secondary indexes in the model schema
     func createIndexStatements() -> String {
         var statement = ""
-        for index in indexes {
-            if case let .index(fields, name) = index, let name = name {
-                statement += CreateIndexStatement(modelSchema: self, fields: fields, indexName: name).stringValue
-            }
+        for case let .index(fields, name?) in indexes {
```

Comment:
can we add a `where` clause and filter out indexes with a `nil` name?

Reply:
That's what the trailing `?` does.

Reply:
Ah, nvm I missed that.